dogpile/cache/backends/redis.py

```diff
@@ -166,26 +170,26 @@ def get_mutex(self, key):
             return None
 
     def get(self, key):
-        value = self.client.get(key)
+        value = self.replica.get(key)
         if value is None:
             return NO_VALUE
         return pickle.loads(value)
 
     def get_multi(self, keys):
         if not keys:
             return []
-        values = self.client.mget(keys)
+        values = self.replica.mget(keys)
```

Comment:
hmm as we are in the generic Redis backend here, these two names might be clearer as "self.writer_client" and "self.reader_client".  this is how we've done it in openstack for other areas.  what do you think?    I can pull this into CI but let me know your appetite for more changes, otherwise I can just take this and finish it up, it looks pretty good.

Reply:
Looks good for coherence, I will change that :-)